packages/wxt/src/core/resolve-config.ts

```diff
@@ -479,14 +481,18 @@ export async function mergeBuilderConfig(
 }
 
 export async function resolveWxtUserModules(
+  rootDir: string,
   modulesDir: string,
   modules: string[] = [],
 ): Promise<WxtModuleWithMetadata<any>[]> {
+  const nodeRequire = createRequire(path.join(rootDir, 'index.js'));
```

Comment:
`index.js` is not required to exist, but we cannot use a directory with `createRequire` or else the directory itself won't be searched